PocketKit/Sources/Sync/RemoteMapping/Item+remoteMapping.swift

```diff
@@ -86,6 +86,38 @@ extension Item {
         givenURL = remote.givenUrl
     }
 
+    func update(from corpusItem: CorpusSlateParts.Recommendation.CorpusItem, in space: Space) {
+        remoteID = corpusItem.id
```

Comment:
I just removed that line, since we don't really need to store `CorpusItem.ID`

Reply:
@Gio2018 is it needed for analytics?

Reply:
@bassrock I might be wrong, but from what I can see in the analytics spreadsheet (not linking it here for security reasons), the entity that's sent over is `CorpusRecommendation` rather than `CorpusItem`?

Reply:
Ok cool!